### PR TITLE
Track integrity of IFRAME SRC= too.

### PR DESCRIPTION
**Pull request by Rudd-O**
IFRAMEs are commonly used to embed PDFs on site pages.

Uploaded PDFs need to be tracked so link integrity is preserved.

EDIT: code updated to bring feature parity with https://github.com/plone/plone.outputfilters/pull/47

### CODE REVIEW

**Comment by Rudd-O** [comment]
@jenkins-plone-org please run jobs

**Comment by Rudd-O** [comment]
This is especially useful for people using RSS / Atom feeds to serve content that embeds PDFs.

**Comment by Rudd-O** [comment]
@jenkins-plone-org please run jobs

**Comment by Rudd-O** [comment]
Tests appear to have a problem, someone added an `import jwt` and that module isn't in the env.

**Comment by Rudd-O** [comment]
Hello?  Anyone?  Thanks in advance.

File: plone/app/linkintegrity/parser.py
Comment (yurj) [inline plone/app/linkintegrity/parser.py:58]: what happen if value is empty or malformed? Can happen?

Comment (Rudd-O) [reply plone/app/linkintegrity/parser.py:58]: https://docs.python.org/3/library/html.parser.html the documentation is unclear, other than saying explicitly that the value is always present (I believe empty tag attribute values are equivalent to the tag attribute being absent).

**Comment by Rudd-O** [comment]
@jenkins-plone-org please run jobs

**Comment by jensens** [comment]
> I'm not entitled to do a proper review, sorry.

Sure, do not underestimate yourself. I would love to reviews from more contributors. If in doubt do not merge and request another approval. In Plone culture, opposed to Zope, the submitter usually does not merge himself, so another pair of eyes is always favorable.

File: plone/app/linkintegrity/parser.py
Comment (jensens) [inline plone/app/linkintegrity/parser.py:47]: this - and above - is a bit lot of the same now.
What about defining two module globals with sets like `TAGS_WITH_SRC = {"img", "source", "audio", ...}` and one with `TAGS_WITH_SRCSET = {...}.and then have two ifs like `if tag in TAGS_WITH_SRC: ....} and same for the other set?

**Review by jensens — CHANGES_REQUESTED** [review]
more DRY would be favorable after more tags are in now.

File: plone/app/linkintegrity/parser.py
Comment (Rudd-O) [reply plone/app/linkintegrity/parser.py]: You're right and it's done.

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Review by jensens — APPROVED** [review]

**Comment by Rudd-O** [comment]
@jenkins-plone-org please run jobs

**Comment by Rudd-O** [comment]
The one test is broken / flaky across multiple projects: https://github.com/plone/Products.CMFPlone/pull/3418

Merging as is.

**Pull request merged**
